Tasks/google-play-release/GooglePlay.ts

```diff
@@ -768,30 +773,51 @@ function resolveGlobPaths(path: string): string[] {
     return [];
 }
 
+function getObbFromParentDirectroy(apkPath: string): string {
+    let obbFile: string = null;
+    let apkDirectory = path.dirname(apkPath);
```

Comment:
Unless I'm mistaken `apkDirectory`, `parentDirectory`, and `filenames` can all be `const`